seatunnel-engine/seatunnel-engine-server/src/main/java/org/apache/seatunnel/engine/server/resourcemanager/AbstractResourceManager.java

```diff
@@ -170,9 +170,19 @@ public CompletableFuture<Void> releaseResource(long jobId, SlotProfile profile)
 
     @Override
     public boolean slotActiveCheck(SlotProfile profile) {
-        return registerWorker.values().stream()
-            .flatMap(workerProfile -> Arrays.stream(workerProfile.getAssignedSlots()))
-            .anyMatch(s -> s.getSlotID() == profile.getSlotID());
+        boolean active;
+        if (registerWorker.containsKey(profile.getWorker())) {
+            active = Arrays.stream(registerWorker.get(profile.getWorker()).getAssignedSlots())
+                .allMatch(s -> s.getSlotID() == profile.getSlotID());
+        } else {
+            active = false;
+        }
+        if (!active) {
+            LOGGER.info("received slot active check failed profile: " + profile);
+        } else {
+            LOGGER.fine("received slot active check success. profile: " + profile);
```

Comment:
remove .

Reply:
> remove .

Thanks, Would it be better add `.` after `received slot active check failed` ?

Reply:
I agree to your suggestion.